test/model_simulation/u0_n_parameter_inputs.jl

```diff
@@ -19,15 +19,15 @@ include("../test_networks.jl")
 # Tests various ways to input u0 and p for various functions.
 let
     test_network = reaction_networks_standard[7]
-    test_osys = convert(ODESystem, test_network)
+    test_osys = complete(convert(ODESystem, test_network))
     @parameters p1 p2 p3 k1 k2 k3 v1 K1 d1 d2 d3 d4 d5
     @species X1(t) X2(t) X3(t) X4(t) X5(t) X6(t) X(t)
 
     for factor = [1e-2, 1e-1, 1e0, 1e1, 1e2, 1e3]
-        u0_1 = factor*rand(rng,length(get_unknowns(test_network)))
```

Comment:
This file have not been properly updated. However, it is really simalr to the indexing test file that I have created to check SII. When I push that PR, that will contain a revamp (and big expansion) of these tests as well. So we will deal with them them

Reply:
ok